src/App.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import CustomTable from './Components/CustomTable';
+import { DetailComponent } from './Components/DetailTable'
 import './App.css';
 import { data, headers } from './Data/Data';
+
 const App: React.FC = () => {
   return (
-    <div>
-      <CustomTable data={data} headers={headers} />
-    </div>
+    <Router>
+      <Routes>
+        <Route path="/" element={<CustomTable data={data} headers={headers} />} />
+        <Route path="/details/:id" element={<DetailComponent />} />
```

Comment:
make a constant folder and in that 
makes a routes.ts file 
then make 
enum APP_ROUTES{
DETAILS_PAGE="/details/:id"
}

Reply:
we will map a route with constants so incase if a route name changes we dont have to go and change everywhere just the constant

Reply:
ok